index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<main id="main">
+  <head> 
+    <meta charset="UTF-8">
+    <title >FCC: Tribute page</title>
+   
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1 id = "title"> Dr. Norman Borlaug </h1>
```

Comment:
I think it would be better to use a header tag here. This will make your code easier to understand.

Reply:
-  Your wordings are very nice. It is polite and suggestive. :+1: 
- [ ] I think , to fit the context of this comment, I think you should write this comment just under `line 11`.